lib/pymedphys/_experimental/streamlit/apps/transfer_check.py

```diff
@@ -70,199 +61,500 @@ def main():
+            ]["Score"].mean()
+        )
+
+    return df
+
+
+def show_dicom(dicom_table):
+    show_table = st.checkbox("View complete DICOM table.")
+    if show_table:
+        st.subheader("DICOM Table")
+        st.dataframe(dicom_table, height=1000)
+
+
+def show_mosaiq(mosaiq_table):
+    show_table = st.checkbox("View complete Mosaiq table.")
+    if show_table:
+        st.subheader("Mosaiq Table")
+        st.dataframe(mosaiq_table, height=1000)
+
+
+def add_alias(dvh_calcs, ALIASES):
+    define_alias = st.checkbox("Define a new structure alias")
+    if define_alias:
+        add_new_structure_alias(dvh_calcs, ALIASES)
+
+
+def add_to_database(constraints_df, institutional_history):
+    add_to = st.checkbox("Add patient to Database")
+    if add_to:
+        add_constraint_results_to_database(constraints_df, institutional_history)
+
+
+def main():
+    server = "PRDMOSAIQIWVV01.utmsa.local"
+    connection = get_cached_mosaiq_connection(server)
+
+    st.sidebar.header("Instructions:")
+    st.sidebar.markdown(
+        """
+    To use this application, you must have the RP file of the plan you want to check. This can be exported in Pinnacle.
+    You will get an error if you select a QA RP file.
+
+    When exporting the DICOM, only the RP is needed for the data transfer check. Once you have that, you can select it where prompted and the application
+    will run.
+
+    To evaluate the DVH, you will need to import the RD and RS files as well.
+    """
+    )
+
+    files = get_patient_files()
+
+    if "rp" in files:
+
+        try:
+            dicom_table = get_all_dicom_treatment_info(files["rp"])
+            dicom_table = dicom_table.sort_values(["field_label"])
+        except AttributeError:
+            st.write("Please select a new RP file.")
+            st.stop()
+
+        mrn = dicom_table.loc[0, "mrn"]
+        mosaiq_table = get_all_treatment_data(connection, mrn)
+        mosaiq_table = drop_irrelevant_mosaiq_fields(dicom_table, mosaiq_table)
+        mosaiq_table = limit_mosaiq_info_to_current_versions(mosaiq_table)
+
+        verify_basic_patient_info(dicom_table, mosaiq_table, mrn)
+        check_site_approval(mosaiq_table, connection)
+        results = compare_to_mosaiq(dicom_table, mosaiq_table)
+        results = results.transpose()
+
+        (
+            field_selection,
+            selected_label,
+            dicom_field_selection,
+        ) = select_field_for_comparison(dicom_table, mosaiq_table)
+        st.subheader("Comparison")
+        if len(selected_label) != 0:
+            show_field_rx(dicom_table, selected_label)
+            check_for_field_approval(mosaiq_table, field_selection, connection)
+            show_comparison_of_selected_fields(dicom_field_selection, results)
+            show_fx_pattern_and_comments(mosaiq_table, field_selection)
+
+        show_dicom(dicom_table)
+        show_mosaiq(mosaiq_table)
 
         if "rs" in files and "rd" in files:
```

Comment:
```suggestion
        if not "rs" in files or not "rd" in files:
            st.stop()
```

Then all the below indenting can be reduced by 1.